pkg/inframetadata/gohai/gohai.go

```diff
@@ -59,13 +64,13 @@ func (m gohaiMarshaler) MarshalJSON() ([]byte, error) {
 // UnmarshalJSON implements the json.Unmarshaler interface.
 // Unmarshals the passed bytes twice (first to a string, then to gohai.Gohai)
 func (m *gohaiMarshaler) UnmarshalJSON(bytes []byte) error {
-	firstUnmarshall := ""
-	err := json.Unmarshal(bytes, &firstUnmarshall)
+	var out string
+	err := json.Unmarshal(bytes, &out)
 	if err != nil {
 		return err
 	}
 
-	err = json.Unmarshal([]byte(firstUnmarshall), &(m.Gohai))
+	err = json.Unmarshal([]byte(out), &(m.Gohai))
```

Comment:
Why not just return this line?
```suggestion
	return json.Unmarshal([]byte(out), &(m.Gohai))
```

Reply:
Done